src/Peachpie.Library/posix.cs

```diff
@@ -74,7 +74,7 @@ public static PhpArray spliti(Context ctx, string pattern, string str, int limit
         /// <param name="useLimit">True if you want to use previous parameter.</param>
         /// <param name="ignoreCase">True if <c>str</c> is matched case insensitive, false otherwise.</param>
         /// <returns>Array containing parts of str</returns>
-        private static PhpArray DoSplit(Context ctx, string pattern, string str, int limit, bool useLimit, bool ignoreCase)
+        public static PhpArray DoSplit(Context ctx, string pattern, string str, int limit, bool useLimit, bool ignoreCase)
```

Comment:
you cannot make a method public just like that - this would introduce a public PHP function `DoSplit`

Reply:
Ah, the reason that is changed to a `public` method is because I was working on some other function that required me to call that. I'm not 100% sure, but I think it was `mb_split`. You can see you guys do the exact same thing in Phalanger [here](https://github.com/DEVSENSE/Phalanger/search?utf8=%E2%9C%93&q=DoSplit&type=). Anyway, for some reason I later removed that function (decided not to implement it for some reason - maybe because I didn't need it). I can definitely change it back to a private method if you want, but maybe you should first see if you want to copy the `mb_string` function from Phalanger, which will need that method to be public. Let me know. Thanks.

Reply:
It should be put in another helper class or be marked with [PhpHidden] attribute; please just ensure there won't be a callable function `DoSplit` after the change.

Reply:
I have reverted to a private access modifier. All should be okay now.